plugin/web/extensions/sidebar/modals/datacatalog/components/content/YourDataPage/LocalDataTab.tsx

```diff
@@ -0,0 +1,114 @@
+import { UserDataItem } from "@web/extensions/sidebar/modals/datacatalog/types";
+import { getExtension } from "@web/extensions/sidebar/utils/file";
+import { Form } from "@web/sharedComponents";
+import { InboxOutlined } from "@web/sharedComponents/Icon/icons";
+import Upload, { UploadProps, UploadFile } from "@web/sharedComponents/Upload";
+import { RcFile } from "antd/lib/upload";
+import { useCallback, useMemo, useState } from "react";
+
+import FileTypeSelect, { FileType } from "./FileTypeSelect";
+
+type Props = {
+  onOpenDetails?: (data?: UserDataItem) => void;
+  setSelectedLocalItem?: (data?: UserDataItem) => void;
+};
+
+const fileFormats = ".kml,.kmz,.csv,.czml,.gpx,.geojson,.json,.zip";
```

Comment:
Can you move these in to the FileTypeSelect and then import here from that file? That way all file type information is in one spot

Reply:
Done!